### PR TITLE
[CIAPP] Add documentation for included/excluded jobs in Jenkins

### PR DESCRIPTION
**Pull request by drodriguezhdez**
### What does this PR do?

This PR adds the instructions about how to configure included/excluded jobs in Jenkins for CI App.

### Preview

https://docs-staging.datadoghq.com/drodriguezhdez/jenkins_include_exclude_pipelines/continuous_integration/setup_pipelines/jenkins/

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/continuous_integration/setup_pipelines/jenkins.md [hunk 1]
```diff
@@ -164,6 +164,27 @@ You can configure the Jenkins Plugin to send custom tags in all pipeline traces:
 **Environment variable**: `DATADOG_JENKINS_PLUGIN_GLOBAL_JOB_TAGS`<br/>
 **Example**: `(.*?)_job_(.*?)_release, owner:$1, release_env:$2, optional:Tag3`
 
+### Including/Excluding pipelines
```
Comment (ruthnaebeck) [inline content/en/continuous_integration/setup_pipelines/jenkins.md:167]:
```suggestion
### Including or excluding pipelines
```

File: content/en/continuous_integration/setup_pipelines/jenkins.md [hunk 2]
```diff
@@ -164,6 +164,27 @@ You can configure the Jenkins Plugin to send custom tags in all pipeline traces:
...
+
+You can configure the Jenkins Plugin to include/exclude some pipelines:
```
Comment (ruthnaebeck) [inline content/en/continuous_integration/setup_pipelines/jenkins.md:169]:
```suggestion
You can configure the Jenkins Plugin to include or exclude some pipelines:
```

File: content/en/continuous_integration/setup_pipelines/jenkins.md [hunk 3]
```diff
@@ -164,6 +164,27 @@ You can configure the Jenkins Plugin to send custom tags in all pipeline traces:
...
+
+1. In the web interface of your Jenkins instance, go to **Manage Jenkins > Configure System**.
+2. Go to the `Datadog Plugin` section, scrolling down the configuration screen.
```
Comment (ruthnaebeck) [inline content/en/continuous_integration/setup_pipelines/jenkins.md:172]:
```suggestion
2. Go to the `Datadog Plugin` section by scrolling down the configuration screen.
```

File: content/en/continuous_integration/setup_pipelines/jenkins.md [hunk 4]
```diff
@@ -164,6 +164,27 @@ You can configure the Jenkins Plugin to send custom tags in all pipeline traces:
...
+3. Click on the `Advanced` button.
+4. Configure the `Excluded Jobs`.
+5. Configure the `Included Jobs`.
+6. Save your configuration.
+
+**Excluded jobs**
```
Comment (ruthnaebeck) [inline content/en/continuous_integration/setup_pipelines/jenkins.md:178]:
```suggestion
#### Excluded jobs
```

File: content/en/continuous_integration/setup_pipelines/jenkins.md [hunk 5]
```diff
@@ -164,6 +164,27 @@ You can configure the Jenkins Plugin to send custom tags in all pipeline traces:
...
+: A comma-separated list of Jenkins jobs that should not be monitored. The exclusion applies to all metrics, traces, events and service checks. Excluded jobs can use regular expressions to reference multiple jobs.<br/>
```
Comment (ruthnaebeck) [inline content/en/continuous_integration/setup_pipelines/jenkins.md:179]:
```suggestion
: A comma-separated list of Jenkins jobs that should not be monitored. The exclusion applies to all metrics, traces, events, and service checks. Excluded jobs can use regular expressions to reference multiple jobs.<br/>
```

File: content/en/continuous_integration/setup_pipelines/jenkins.md [hunk 6]
```diff
@@ -164,6 +164,27 @@ You can configure the Jenkins Plugin to send custom tags in all pipeline traces:
...
+**Environment variable**: `DATADOG_JENKINS_PLUGIN_EXCLUDED`<br/>
+**Example**: `susans-job,johns-.*,prod_folder/prod_release`
+
+**Included jobs**
```
Comment (ruthnaebeck) [inline content/en/continuous_integration/setup_pipelines/jenkins.md:183]:
```suggestion
#### Included jobs
```

File: content/en/continuous_integration/setup_pipelines/jenkins.md [hunk 7]
```diff
@@ -164,6 +164,27 @@ You can configure the Jenkins Plugin to send custom tags in all pipeline traces:
...
+: A comma-separated list of Jenkins job names that should be monitored. If the included jobs are empty, all jobs that are not excluded explicitly will be monitored. The inclusion applies to all metrics, traces, events and service checks. Included jobs can use regular expressions to reference multiple jobs.<br/>
```
Comment (ruthnaebeck) [inline content/en/continuous_integration/setup_pipelines/jenkins.md:184]:
```suggestion
: A comma-separated list of Jenkins job names that should be monitored. If the included jobs list is empty, all jobs that are not excluded explicitly are monitored. The inclusion applies to all metrics, traces, events, and service checks. Included jobs can use regular expressions to reference multiple jobs.<br/>
```

**Review by ruthnaebeck — CHANGES_REQUESTED** [review]

File: content/en/continuous_integration/setup_pipelines/jenkins.md (hunk 4 above)
Comment (drodriguezhdez) [reply content/en/continuous_integration/setup_pipelines/jenkins.md:178]: About this, I'm using the same approach used in the previous sections. Should we use a different approach here?

Comment (ruthnaebeck) [reply content/en/continuous_integration/setup_pipelines/jenkins.md:178]: ok, we can skip this for now. Headers just make it easier to navigate the page.

**Review by ruthnaebeck — APPROVED** [review]

**Pull request merged**